core/templates/pages/signup-page/modal-templates/registration-session-expired-modal.controller.spec.ts

```diff
@@ -29,6 +31,10 @@ describe('Registration Session Expired Modal Controller', function() {
   };
 
   beforeEach(angular.mock.module('oppia', function($provide) {
+    var ugs = new UpgradedServices();
```

Comment:
@srijanreddy98 is this ok?

Reply:
No, @orthodoxparadox we don't use ugs anymore. Please remove it.